book/api/websocket.md

```diff
@@ -587,21 +587,21 @@ potential underflow.
 | Field               | Type                | Description
 |---------------------|---------------------|------------
 | next_leader_slot    | `number\|null`      | The next leader slot |
-| tile_primary_metric | `TilePrimaryMetric` | Per-tile-type primary metrics.  Some of these are point-in-time values (P), and some are aggregated since the end of the previous leader slot (A) |
+| tile_primary_metric | `TilePrimaryMetric` | Per-tile-type primary metrics.  Some of these are point-in-time values (P), and some are 1-second moving window averages (W) |
 
 **`TilePrimaryMetric`**
 | Field   | Type     | Description |
 |---------|----------|-------------|
-| net_in  | `number` | Ingress bytes per second (P) |
+| net_in  | `number` | Ingress bytes per second (W) |
 | quic    | `number` | Active QUIC connections (P) |
-| verify  | `number` | Fraction of transactions that failed sigverify (A) |
-| dedup   | `number` | Fraction of transactions deduplicated (A) |
+| verify  | `number` | Fraction of transactions that failed sigverify (W) |
+| dedup   | `number` | Fraction of transactions deduplicated (W) |
 | pack    | `number` | Fraction of pack buffer filled (P) |
-| bank    | `number` | Execution TPS (P) |
-| poh     | `number` | Fraction of time spent hashing (P) |
-| shred   | `number` | Shreds processed per second (P) |
-| store   | `number` | 50% percentile latency (A) |
-| net_out | `number` | Egress bytes per second (P) |
+| bank    | `number` | Execution TPS (W) |
+| poh     | `number` | Fraction of time spent hashing (W) |
```

Comment:
Looks like we don't send a few of these (e.g. poh, shred, ..) remove them here I think